src/OpenTelemetry.Sampler.AWS/AWSXRayRemoteSamplerBuilder.cs

```diff
@@ -70,7 +70,9 @@ public AWSXRayRemoteSamplerBuilder SetEndpoint(string endpoint)
     /// <returns>an instance of <see cref="Trace.Sampler"/>.</returns>
     public Trace.Sampler Build()
     {
+#pragma warning disable CA2000
```

Comment:
nit: I think it's a good idea to include the title of what is being disabled.

```suggestion
#pragma warning disable CA2000 // Dispose objects before losing scope
```